src/main/java/com/redhat/jenkins/plugins/bayesian/Bayesian.java

```diff
@@ -211,7 +298,7 @@ public void setUrl(String url) {
     }
     
     public String getGitUrl() {
-        return gitUrl;
+       return gitUrl;
```

Comment:
pls. use 4 spaces (this might be some form of typo)

Reply:
yes.. typo.. rectified